modules/metastore/config/install/metastore.settings.yml

```diff
@@ -1,21 +1,21 @@
 property_list:
   'theme': 'theme'
   'keyword': 'keyword'
-  'publisher' : 'publisher'
+  'publisher': 'publisher'
   'distribution': 'distribution'
-  'contactPoint': 0
-  '@type': 0
-  'title': 0
-  'identifier': 0
-  'description': 0
-  'accessLevel': 0
-  'accrualPeriodicity': 0
-  'describedBy': 0
-  'describedByType': 0
-  'issued': 0
-  'license': 0
-  'modified': 0
-  'references': 0
-  'spatial': 0
-  'temporal': 0
-  'isPartOf': 0
+  'contactPoint': '0'
+  '@type': '0'
+  'title': '0'
+  'identifier': '0'
+  'description': '0'
+  'accessLevel': '0'
+  'accrualPeriodicity': '0'
+  'describedBy': '0'
+  'describedByType': '0'
+  'issued': '0'
+  'license': '0'
+  'modified': '0'
+  'references': '0'
+  'spatial': '0'
+  'temporal': '0'
+  'isPartOf': '0'
```

Comment:
Why all the single quotes? Only @type needs them.

Reply:
They're all strings. https://github.com/GetDKAN/dkan/blob/2.x/modules/metastore/config/schema/metastore.schema.yml#L15 If we need to change them to be something else then we can.